### PR TITLE
as requested, reference to #1578 #1588

### PR DESCRIPTION
**Pull request by piengeng**
copy paste from https://raw.githubusercontent.com/grpc-ecosystem/grpc-gateway/f3b7d47220131ec67b59d0e44539d58cc76ac61e/docs/_docs/grpcapiconfiguration.md to v2 branch as requested.

### CODE REVIEW

File: docs/_docs/grpcapiconfiguration.md [hunk 1]
```diff
@@ -64,4 +64,11 @@ The following is equivalent to the basic [usage example](usage.html) but without
    
    This will generate a reverse proxy `gen/go/your/service/v1/your_service.pb.gw.go` that is identical to the one produced for the annotated proto.
 
+6. Generate the optional your_service.swagger.json
+
+    ```sh
+    protoc -I. --swagger_out=grpc_api_configuration=path/to/your_service.yaml:./gen/go \
```
Comment (johanbrandhorst) [inline docs/_docs/grpcapiconfiguration.md:70]: We renamed `protoc-gen-swagger` to `protoc-gen-openapiv2` in v2:
```suggestion
    protoc -I. --openapiv2_out=grpc_api_configuration=path/to/your_service.yaml:./gen/go \
```

Comment (piengeng) [reply docs/_docs/grpcapiconfiguration.md:70]: Output is still suffix swagger.json?

Comment (johanbrandhorst) [reply docs/_docs/grpcapiconfiguration.md:70]: Yes

Comment (johanbrandhorst) [reply docs/_docs/grpcapiconfiguration.md:70]: This has not been resolved still?

**Comment by johanbrandhorst** [comment]
It would be really nice to have this in the v2 docs, why did you close it? Any chance I could convince you to reopen it?

**Pull request closed without merging**